tower_of_london/config.json

```diff
@@ -6,8 +6,10 @@
                 "style.css",
                 "static/js/jspsych/custom_plugins/jspsych-single-stim-button.js"
                ],
-        "tag": "tower_of_london",
-        "cognitive_atlas_task_id": "",
+
+        "tag": "tol",
```

Comment:
This should be "tower_of_london" and not "tol" - the task tag must be equivalent to the folder name